doc/coding_standards.rst

```diff
@@ -88,17 +88,17 @@ standards:
     [1, 2, 3]
     {'name': 'Fabien'}
 
-* Do not put any spaces before and after ``=`` in macro argument declarations:
+* Put exactly one space before and after ``=`` in macro argument declarations:
 
   .. code-block:: twig
 
-    {% macro html_input(class="input") %}
+    {% macro html_input(class = "input") %}
 
-* Put exactly one space after the ``:`` sign in macro argument declarations:
+* Put exactly one space after the ``:`` sign when using named arguments:
```

Comment:
I'm not sure it's worth mentioning a standard for named argument with `=` since 
```
Use : instead of = to separate argument names and values:

{{ data|convert_encoding(from: 'iso-2022-jp', to: 'UTF-8') }}
```